### ISSUE TITLE
page detection doesn't work correctly in some case

### ISSUE THREAD

**Issue by crapthings** [Breta01/handwriting-ocr#91]
i've got these result

![-7ndqgxkSUrzGVFtFaQRJ](https://user-images.githubusercontent.com/1147704/58317408-4a408c00-7e48-11e9-8cb5-c412db1f4325.jpeg)
![5XwnxdWwHBDxlL10dAz3x](https://user-images.githubusercontent.com/1147704/58317410-4d3b7c80-7e48-11e9-8fed-a182871ef894.jpeg)
![hdYyCIWhcJE__IdJjn9ha](https://user-images.githubusercontent.com/1147704/58317423-4f9dd680-7e48-11e9-8a5c-879c2c05b538.jpeg)
![5mglg9DLQ5eh2vniGWLag](https://user-images.githubusercontent.com/1147704/58317430-52003080-7e48-11e9-9e59-25f9e721d70a.jpeg)

**Comment by crapthings** [comment]
![R3xcsIhqSFFfynf3AQwD7](https://user-images.githubusercontent.com/1147704/58317537-8d026400-7e48-11e9-8634-0a178bab337f.png)
![Vdp_PNleojuNqm9dDSqX2](https://user-images.githubusercontent.com/1147704/58317539-8d026400-7e48-11e9-8758-d60110bd327f.png)

this is image from example
https://github.com/Breta01/handwriting-ocr/blob/master/notebooks/page_detection.ipynb

**Comment by crapthings** [comment]
what if we use object detective to get object bounding then work in that bounding. will this work better?

**Comment by Breta01** [comment]
Yes, if you train it on some good dataset, I guess it will work better. Especially in this situations when there is more complex background. I haven't tried it by myself yet.
